pkg/orchestrator/etcd_worker.go

```diff
@@ -261,6 +276,29 @@ func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session,
 			if !rl.Allow() {
 				continue
 			}
+
+			// we need to check if this the owner key is delete before we tick the reactor
+			if role == pkgutil.RoleOwner.String() {
+				var id model.CaptureID
+
+				err = retry.Do(ctx, func() error {
+					id, err = worker.client.GetOwnerID(ctx)
```

Comment:
Is `GetOwnerID` already retried internally? If yes, it looks that no need to retry here ?